deployments/deploy-ssh.sh

```diff
@@ -34,7 +34,7 @@ if [ -z "${SPEC_DIR}" ]; then
 fi
 
 if [ -z "${SCHEDULER}" ]; then
-    SCHEDULER="k8s"
+    SCHEDULER="rke"
```

Comment:
Should we be updating this?

Reply:
No no. Will remove this before merging

Reply:
@snigdha-px Please add a comment so that we don't forget it.

Reply:
sure

Reply:
@snigdha-px Please do the needful here before merging.

Reply:
Probably the checks are failing because of this.